recipes/facebook_business/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "facebook_business" %}
+{% set version = "9.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/facebook_business-{{ version }}.tar.gz
+  sha256: 3409103f0b66abc1e1b0d76679151672618dfdf0491223cd0e6a02ad59d43c9a
+
+build:
+  number: 0
```

Conda-forge doesn't build for Python <3.6 anymore so I think this can be noarch:
```suggestion
  noarch: python
  number: 0
```